lib/bin/command-proxy-client.js

```diff
@@ -10,44 +10,43 @@
 // ----- OUTPUT -----
 // <OUTPUT>
 
-"use strict";
+import domain from 'domain';
+import net from 'net';
 
-var output = '';
+let output = '';
 
-function exit(status) {
+function exit (status) {
   console.log('----- OUTPUT -----');
   console.log(output);
   // chill out before exiting
-  process.nextTick(function () {
+  process.nextTick(() => {
     process.exit(status);
   });
 }
 
-var d = require('domain').create();
+let d = domain.create();
```

Comment:
i'm not sure we want to use domains anymore, they've sort of been deprecated by the node folks. IMO putting one try/catch at the top level of the `connect` function would have the same effect